internal/loglevel/loglevel.go

```diff
@@ -0,0 +1,51 @@
+// SPDX-FileCopyrightText: 2023 Comcast Cable Communications Management, LLC
```

Comment:
It's 2024.

Reply:
I'll update all the other files in a separate commit.  most of the repo is 2023